api/handlers/listhandler.py

```diff
@@ -61,6 +62,13 @@ def initialize_list_configurations():
             'storage_schema_file': 'note.json',
             'input_schema_file': 'note.json'
         },
+        'analyses': {
+            'storage': liststorage.AnalysesStorage,
+            'permchecker': listauth.default_sublist,
+            'use_object_id': True,
+            'storage_schema_file': 'analysis.json',
+            'input_schema_file': 'analysis.json'
```

Comment:
Good catch. The data model for analyses is specified [here]( https://github.com/scitran/core/wiki/Data-Model,-v3#analysis-draft), but it's still a draft. 

To avoid blocking clients, I didn't add the schema.
I will add one though.